attention_package/FoveatedImage[] foveated_images
